### ISSUE TITLE
Support Anchor / Hash URL parts

### ISSUE THREAD

**Issue by Myster** [tmenier/Flurl#29]
Example code:
```
var baseUrl = "?q=term&page=2#results"
var output = baseUrl.SetQueryParam("page",3).ToString();
```
Actual output:
```
"?q=term&page=3"
```
Expected output
```
"?q=term&page=3#results"
```

**Comment by tmenier** [comment]
Thanks for reporting. I will see about getting that working soon.

**Comment by tmenier** [comment]
Fixed for Flurl 2.0, which will be released soon. Flurl.Url will include a Fragment property.

**Comment by tmenier** [comment]
Test: https://github.com/tmenier/Flurl/blob/master/Test/Flurl.Test.Shared/UrlBuilderTests.cs#L306-L314
